### ISSUE TITLE
is this project active?

### ISSUE THREAD

**Issue by ccl0326** [dhilipkumars/MrRedis#67]

**Comment by mqasimsarfraz** [comment]
Same concern. Is the project active?

**Comment by dhilipkumars** [comment]
Hi @ccl0326 @MQasimSarfraz ,

We do not have bandwidth to lookafter this project and actively maintaint in for now, we have been inviting contributors for this project.  We would quickly review and merge any new PR that the community raises. 

Feel free to raise any PR. 

I think DC/OS is comming up with a project/scheduler to support redis in collaboration with redis-labs.  Not sure about that projects status. 

The next logical step for this project is to re-write it (if it only takes minimal effort and resuse most of the components) using DCOS-SDK and donate it to the community as it will be in written in a uniform framework and will be easy to maintain it aswell.

**Comment by mqasimsarfraz** [comment]
@dhilipkumars  Many thanks for response. I was looking for features like DC/OS VIPs and it turns out there were open issues around it already but seems there is work pending on it. But having said that I will be happy to contribute as much as I can.

**Comment by ccl0326** [comment]
@dhilipkumars Thanks for your reply. :)
